README.md

```diff
@@ -29,9 +30,11 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Armis](http://www.armisgroup.com/) [:rocket:](http://www.armisgroup.com/ofertas-de-trabalho/) | Technology development and consultancy. | `Lisboa` `Porto` |
 | [blockbird Ventures](https://www.blockbird.ventures) [:rocket:](https://blockbird.ventures/careers/) | Blockchain and Ethereum development. | `Lisboa` `Remote` |
 | [BOLD International](https://www.boldint.pt/) [:rocket:](https://www.boldint.pt/carreiras/) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
+| [Bool Software](https://www.bool.pt/) | Software design, development, training and consultancy company specialized in OutSystems technology. | `Lisboa` <br>`Açores` |
```

Comment:
Can you please shorten the description?  
Also, can you please replace Açores with the city itself?

Reply:
Sure, i'll do that